src/Console/InfectionContainer.php

```diff
@@ -257,7 +257,7 @@ public function buildDynamicDependencies(InputInterface $input): void
         $this['coverage.checker'] = static function () use ($input): CoverageRequirementChecker {
             return new CoverageRequirementChecker(
                 \strlen(trim($input->getOption('coverage'))) > 0,
-                $input->getOption('initial-tests-php-options')
+                (string) $input->getOption('initial-tests-php-options') ?? ''
```

Comment:
Due to how php prioritises casts, this statement is actually as follows:
```php
((string) $input->getOption('initial-tests-php-options')) ?? ''
```
While the desired way would be:
```php
 (string) ($input->getOption('initial-tests-php-options') ?? '')
```

So i'd suggest adding parenthesis here to help

```suggestion
                (string) ($input->getOption('initial-tests-php-options') ?? '')
```